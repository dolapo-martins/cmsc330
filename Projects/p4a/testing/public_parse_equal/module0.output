Fun(Type_Int,"main",List[],List[Assign(Id "a",Sum(Id "a",Sum(Mult(Id "b",Paren(Sum(Pow(Paren(Sum(Id "c",Id "e")),Id "f"),Id "d"))),Mult(Pow(Num 6,Num 4),Num 5))))])
