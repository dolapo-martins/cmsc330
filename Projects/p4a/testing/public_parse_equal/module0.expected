Fun(Type_Int,"main",List[],List[Define(Type_Int,Id "a");Assign(Id "a",Num 100);If(Equal(Id "a",Num 100),List[Assign(Id "a",Num 200);Print(Id "a")],List[])])
